simpledotorg/simple-server
# Progress tab period report template

**Story card:** [sc-7272](http://localhost:3000/reports/progress/uhc-mace-village)

## Because

We're creating new period subpages for the Progress Tab. This PR creates a single HTML template for the daily, monthly, and yearly report pages.

## This addresses
![image](https://user-images.githubusercontent.com/16785131/153605604-5ba8bb42-8bea-49cb-8a8a-dd945b84a4ad.png)

+ Creates a new `_period_report.html.erb` partial
+ Adds dummy data to the controllers (will be deleted when real data is available)
+ Creates a new `progress_tab.js` standalone script to show/hide the report pages

## Test instructions

+ Enable the `:new_progress_tab` Flipper flag
+ Go to a facility report page and click "Progress reports"
+ Click through the "Daily", "Monthly", and "Yearly" reports. Make sure that the links work, the UI looks good.
+ When you click the "back" arrow, the Home page should appear. Scroll to the very bottom of the home page and make sure just the home page is showing.
+ Open `.env.development` and set `DEFAULT_COUNTRY=ET`
+ Restart your server
+ Go to a facility report page, click "Progress reports" and open the "Daily" report. There shouldn't be a yellow banner at the end of the page (should only appear for India)
